cmd/detectExecuteScan.go

```diff
@@ -2,6 +2,7 @@ package cmd
 
 import (
 	"fmt"
+	"github.com/SAP/jenkins-library/pkg/piperutils"
```

Comment:
I would suggest to alias this as `SliceUtils` to have a move descriptive code:
```suggestion
	"sliceUtils github.com/SAP/jenkins-library/pkg/piperutils"
```